### DISCUSSION TITLE
Is it possible to set a function for how long sessile organisms will survive after habitat turning unsuitable?

### DISCUSSION THREAD

**Discussion by duriber** [RangeShifter/RangeshiftR-tutorials#10] [Q&A] [answered]
Hi RangeShiftR team,

I have been playing with the R package to simulate range dynamics and realised that after a change in the landscape (using dynamic landscapes), all the individuals that were living in a place that was suitable but turned non-suitable die the year that the landscape changes. I would like to know whether it is possible to relax that assumption to allow the individuals in an unsuitable habitat to continue to be alive for a  predefined period of time (or even better, to implement a function of survival probability that decays with time since the habitat turned non-suitable). Is something like that already possible to implement? if not, is there a way around it? 

Thank you very much for implementing the software into an R package. 
David.

**Comment by JetteReeg** [comment, marked as answer]
Hi David, 
for an unstructured population, you currently only have the option that populations die if a patch is destroyed. However, in stage-structured populations you can allow dispersal directly after the destruction (see manual & the R help for the StageStructure function).
Best, 
Jette
